--- a/amd/amdgpu/amdgpu_ttm.c.orig	2017-01-28 03:09:32.173714601 +0100
+++ b/amd/amdgpu/amdgpu_ttm.c	2017-01-28 03:10:26.053810902 +0100
@@ -255,7 +255,11 @@
 
 	if (amdgpu_ttm_tt_get_usermm(bo->ttm))
 		return -EPERM;
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 9, 0)
+	return drm_vma_node_verify_access(&abo->gem_base.vma_node, (struct drm_file*)filp);
+#else
 	return drm_vma_node_verify_access(&abo->gem_base.vma_node, filp);
+#endif
 }
 
 static void amdgpu_move_null(struct ttm_buffer_object *bo,
