From 37c88d4aab244a86e706a32377494aaf80b7aa65 Mon Sep 17 00:00:00 2001
From: Mai Lavelle <mai.lavelle@gmail.com>
Date: Fri, 2 Dec 2016 20:30:08 -0500
Subject: [PATCH 2/2] silence indentation build error

---
 amd/dal/modules/freesync/freesync.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/amd/dal/modules/freesync/freesync.c b/amd/dal/modules/freesync/freesync.c
index b787045..6a13196 100644
--- a/amd/dal/modules/freesync/freesync.c
+++ b/amd/dal/modules/freesync/freesync.c
@@ -501,7 +501,7 @@ static bool set_freesync_on_streams(struct core_freesync *core_freesync,
 
 				/* Program only if v_total_nominal is in range*/
 				if (v_total_nominal >=
-					streams[stream_idx]->timing.v_total)
+					streams[stream_idx]->timing.v_total) {
 
 					/* Update the freesync context for
 					 * the stream
@@ -509,6 +509,7 @@ static bool set_freesync_on_streams(struct core_freesync *core_freesync,
 					update_stream_freesync_context(
 						core_freesync,
 						streams[stream_idx]);
+				}
 
 					core_freesync->dc->stream_funcs.
 					adjust_vmin_vmax(
-- 
2.10.2

